src/forum-tool-selector-settings/ForumToolSelectorSettings/ForumToolSelectorContainer/ForumSelectorRow.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import PropTypes from 'prop-types';
+import ForumCardDeck from './ForumSelectorRow/ForumCardDeck';
+import FeaturesTable from './ForumSelectorRow/FeaturesTable';
+
+export default function ForumSelectorRow({
+  forumList, featuresList, selectedForumName, onSelected, onUnselected,
+}) {
+  return (
+    <div className="mt-4">
+      <ForumCardDeck
+        page={forumList}
+        selectedId={selectedForumName}
```

Comment:
Why refer to the same thing by different names in different components?
```suggestion
        selectedId={selectedId}
```